regression/cbmc-java/war-file1/url.txt

```diff
@@ -0,0 +1 @@
+https://tomcat.apache.org/tomcat-6.0-doc/appdev/sample/
```

Comment:
I very much appreciate that comment - but you might even place it in the test.desc file (so as not to create a non-standard file): you can place comments after another `--`, see `test.pl --help`